### PR TITLE
Add proxy for snapweb

### LINKED ISSUE #79: Add proxy to the snapweb app store
Just like for Spreed.ME, we have to add a proxy to the snapweb snap, but this time it's mandatory.

### PR DESCRIPTION
**Pull request by oparoz**
This makes it possible to integrate the snap store inside Nextcloud

Fixes #79

### CODE REVIEW

File: src/apache/conf/ssl.conf [hunk 1]
```diff
@@ -161,3 +161,17 @@ SSLRandomSeed connect file:/dev/urandom 512
         Header always set Strict-Transport-Security "max-age=63072000; includeSubdomains;"
     </IfDefine>
 </VirtualHost>
+
+#########################
+#        Snapweb        #
+#########################
+Listen 4201
+<VirtualHost *:4201>
```
Comment (kyrofa) [inline src/apache/conf/ssl.conf]: Obviously snapweb is already listening (with no SSL) on 4200. What is the purpose of also having it here? Just to add SSL to it?

Comment (oparoz) [inline src/apache/conf/ssl.conf]: Without SSL support, we cannot integrate it inside Nextcloud, so we need to create a virtualhost on our reverse proxy.

Comment (kyrofa) [inline src/apache/conf/ssl.conf]: This also means one can only use snapweb from within nextcloud if SSL is enabled, right? Isn't that a little odd?

Comment (oparoz) [inline src/apache/conf/ssl.conf]: No, you have to use 2 different URLs, one for SSL and one for without SSL. IIRC, only the port changes.

Comment (kyrofa) [inline src/apache/conf/ssl.conf]: Note that Snapweb has gained HTTPS. This probably needs to be re-evaluated.

Comment (oparoz) [inline src/apache/conf/ssl.conf]: Yep, I'd say it's not needed any more for most users if they can simply add the URL. Enterprise users might need to create their own vhost, but it's out of sope for this project.

**Pull request closed without merging**